src/app/dataset/dataset.component.html

```diff
@@ -8,6 +8,9 @@
   }"
 ></app-load-animation>
 
+<app-reportsimple [openReportSimple]="reportMsg" (closeReportSimple)="clearReportMsg()">
```

Comment:
If I format it the way suggested here and run:
  `npm run fix`
then it puts the code back the way it was (because the single line is already short enough)